internal/host/hostcommands/install_cmd.go

```diff
@@ -350,6 +350,9 @@ func getDHCPArgPerNIC(network *net.IPNet, nic *models.Interface, ipv6 bool, host
 		return nil, err
 	}
 	if found {
+		if dualStack && !ipv6 {
```

Comment:
I think in case of dual stack the primary machine network is always IPv4.  So the checking for IPv6 is not needed.
Even if it was IPv6, I think you would still need this parameter.

Reply:
It was just in case